Nodsoft.WowsReplaysUnpack.FileStore/Definitions/FileSystemDefinitionLoaderOptions.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Nodsoft.WowsReplaysUnpack.FileStore.Definitions;
+
+/// <summary>
+/// Defines settings for the <see cref="FileSystemDefinitionLoader"/>.
+/// </summary>
+public record FileSystemDefinitionLoaderOptions
```

Comment:
I know you love records, but don't you remember when we had a chat about it?
If you don't need it's special features (like equality and immutability) it actually adds more overhead than a normal class.
Not that it really matters for an options object that's only there once. But you know, gotta mention it :D